app/notification/magic_link/map_contents.py

```diff
@@ -23,8 +28,16 @@ def from_json(json_data: dict):
         content = MagicLink.process_data(json_data).get("content")
         return MagicLink(
             contact_info=data.contact_info,
-            fund_name=content.get("fund_name"),
-            magic_link=data.content.get("magic_link_url"),
+            fund_name=content.get(Config.NOTIFICATION_MAGIC_LINK_FUND_NAME),
```

Comment:
Should the fund name be dynamic rather than from config?

Reply:
It is - this is just the field name in the json that comes from the config

Reply:
I think the naming made it a bit confusing. Maybe these should be called e.g.  `NOTIFICATION_MAGIC_LINK_FUND_NAME_FIELD` etc?

Reply:
We've now changed them to this pattern: `NotifyConstants.FIELD_xyz`